spec/services/atmosphere/optimizer_spec.rb

```diff
@@ -96,18 +98,18 @@
           appl_type.preference_memory = biggest_os_flavor.memory
           appl_type.save
 
-          tmpl, flavor = subject.select_tmpl_and_flavor([tmpl_at_amazon, tmpl_at_openstack])
+          tmpl, tenant, flavor = subject.select_tmpl_and_flavor_and_tenant([tmpl_at_amazon, tmpl_at_openstack])
 
           expect(flavor).to eq optimal_flavor
           expect(tmpl).to eq tmpl_at_amazon
         end
 
-        context 'preferences exceeds resources of avaiable flavors' do
+        context 'preferences exceeds resources of available flavors' do
           it 'returns nil flavor' do
             appl_type.preference_cpu = 64
             appl_type.save
 
-            tmpl, flavor = subject.select_tmpl_and_flavor([tmpl_at_amazon, tmpl_at_openstack])
+            tmpl, tenant, flavor = subject.select_tmpl_and_flavor_and_tenant([tmpl_at_amazon, tmpl_at_openstack])
```

Comment:
Useless assignment to variable - `tmpl`. Use `_` or `_tmpl` as a variable name to indicate that it won't be used.<br>Useless assignment to variable - `tenant`. Use `_` or `_tenant` as a variable name to indicate that it won't be used.<br>Line is too long. [113/80]